src/categories/categories.service.ts

```diff
@@ -0,0 +1,51 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { log } from 'console';
+import { AppModule } from 'src/app.module';
+import { Repository } from 'typeorm';
+import { Category } from './Category';
+
+@Injectable()
+export class CategoriesService {
+    constructor(@InjectRepository(Category) private categoryRepository: Repository<Category>){}  
+    categories:Category[]
+
+    async createCategory(item:Category){
+        const {name , icon , color} = item
+        const category = await this.categoryRepository.save({name , icon , color})
```

Comment:
why not just save the item?
`const category = await this.categoryRepository.save(item)`
This is a question, not a review comment

Reply:
there was no specific reason